x/ccv/consumer/keeper/keeper.go

```diff
@@ -26,7 +30,7 @@ import (
 type Keeper struct {
 	storeKey          sdk.StoreKey
 	cdc               codec.BinaryCodec
-	paramSpace        paramtypes.Subspace
+	paramstore        paramtypes.Subspace
```

Comment:
`paramSpace` and `accountKeeper` were renamed to match the stakingKeeper's naming conventions